src/util/globalize.ts

```diff
@@ -75,5 +78,5 @@ export function globalizeDelegator<T, O, R>(method: string, args: DelegatorOptio
 	}
 
 	const globalize = getGlobalize(locale);
-	return (<any> globalize)[method].apply(globalize, methodArgs);
+	return (<any>globalize)[method].apply(globalize, methodArgs);
```

Comment:
I'm surprised the lack of space after `<any>` doesn't cause problems when running `grunt`. Elsewhere we've had to change `<any> value` to `(value as any)`.

Reply:
why would it cause problems running `grunt`?